src/common/exponent/xdlInterface.ts

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {CommandExecutor, CommandVerbosity} from "../commandExecutor";
+import {HostPlatform} from "../hostPlatform";
+import {Log} from "../log/log";
+
+import * as XDLPackage from "xdl";
+import * as path from "path";
+import * as Q from "q";
+
+const XDL_VERSION = "0.12.0";
```

Comment:
This version should be bumped up, I believe 0.15 has some improved functionality, but it's up to 0.19 now so probably go with that if it works.